sky/compositor/compositor_options.cc

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "sky/compositor/compositor_options.h"
+
+#include <mutex>
+
+namespace sky {
+namespace compositor {
+
+CompositorOptions& CompositorOptions::Shared() {
+  static std::once_flag once;
```

Comment:
does the standard promise that this object has no destructor?  if multiple threads enter this concurrently are you sure that the once_flag itself will be initialized correctly (see comment below about magic statics)